packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,223 @@
+
+## Responsive Design
+
+The ApplicationLayout has two rendering modes: `standard` and `compact`.
+  - The `standard` rendering occurs at `medium`, `large`, and `huge` breakpoints.
+  - The `compact` rendering occurs at `tiny` and `small` breakpoints.
+
+Depending on the rendering mode, the ApplicationLayout's prop data is rendered in different ways.
+
+|Prop|`standard` Rendering|`compact` Rendering|
+|---|---|---|
+|`nameConfig`|Content is rendered within ApplicationLayout's header.|Same as `standard`.|
+|`utilityConfig`|Content is rendered within the ApplicationLayout's header. The utility menu is presented using a popup (as provided by `terra-popup`).|Content is rendered within the ApplicationMenuWrapper that wraps all menu content at `compact` breakpoints. The utility menu is presented modally (using the ApplicationLayout's included ModalManager).|
+|`navigationItems`|Content is rendered within the ApplicationLayout's header as tabs (as provided by `terra-application-links`).|Content is rendered within a custom menu generated by the ApplicationLayout. This menu is injected into the `routingConfig` for the `'/'` path so that it will preceed any other defined routes. Other menu components are able to navigate to the default navigation menu by calling their `routingStackDelegate`'s `showParent` function.|
+
+## Menu/Content Component Configuration
+
+The `routingConfig` prop specifies which components will be rendered in the ApplicationLayout's `content` and `menu` regions. These components receive a set of props from the ApplicationLayout that they must utilize; otherwise, these components can be completely defined by the consuming application.
+
+> Note: Because the `ApplicationLayout` utilizes client-side routing provided by `react-router-dom`, the usage of `<a href=>` links and page anchors is restricted. Usage of links and page anchors may negatively impact navigation within an application. `<Link />` components provided by `react-router-dom` or custom events can and should be used instead.
+
+The components rendered in these regions will receive the following props from the ApplicationLayout:
+
+### `app`
+
+The AppDelegate instance (as `app`) provided to the ApplicationLayout's contents is created by ApplicationLayout's default `ModalManager`.
+
+Please review the AppDelegate documenation with `terra-app-delegate` for more information regarding the available disclosure options.
+
+### `layoutConfig`
+
+The `layoutConfig` contains properties describing the ApplicationLayout's state and functions that allow for the manipulation of that state.
+
+Please review the `Layout` component in `terra-layout` for more information.
+
+### `routingStackDelegate`
+
+The `routingStackDelegate` contains APIs that allow for virtual navigation within a set of matched component paths. Please review the RoutingStackDelegate documentation within `terra-navigation-layout` for more information.
+
+## Summary
```

Comment:
I like this part of the readme. Is there anyway we can create a summary example, and put it after this section, or link to a new MD file with the example?